### DISCUSSION TITLE
How to deploy metadata changes (fields collections and other configurations)

### DISCUSSION THREAD

**Discussion by aviNrich** [directus/directus#6947] [Archive]
Hi,
I'm new to directus, I want to examine it for our next project.
Most of the important configuration is placed on the database. so lets say that in my local dev env I have added new fields, changed some roles, changed the display...
How can I deploy these changes? I don't want to run these kind of changes directly in production env.
I know that I can migrate my source tables (articles, posts) via [custom migration](https://docs.directus.io/guides/migrations/#custom-migrations), but how do deploy other collection settings changes? it is not part of my git...

Thanks

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/3891
